js/utils/utils.js

```diff
@@ -281,8 +281,12 @@ Util.updateGroups = function (groupType, ids, selector) {
 				url: apiPath + "metadata",
 				data: JSON.stringify(metadata),
 				async: true,
-				success: function () {
-					location.reload();
+				success: function (report) {
+					if (report.status == "ERROR") {
```

Comment:
@SferaDev This code is not working for me. 'report' variable is a string and needs to be parsed to JSON. Have you ever tested this code? I am tempted to change it but I would like to understand if this code actually makes sense in any other case

Reply:
We did this changes on-the-fly in a hangouts with Nacho but yes we did test it and seemed to work properly...

I will test it again and report back

Reply:
Finally, it was a strange error that may have to do with the browser version or OS. Seems to be fixed with https://github.com/EyeSeeTea/datadictionary/pull/94/commits/ad587d4774df62d93905616ec4a180c4f4c4a6ca (forcing the conversion on the request)